src/global_vars.py

```diff
@@ -1 +1,7 @@
+import os
+
 APPLICATION_DIR = "/etc/ingestation"
+
+options = {
```

Comment:
I don't think this dict is required atm, Conf.get_conf_path() is doing precisely that.

Reply:
In the end, we will enable passing project and operator confs as cli arguments, application will never be a user option.